Create PoS General FAQ

animedow opened the pull request:
A simplified general FAQ for non technical people who are new to Decred.
Based on the most frequently asked questions in the slack channel.

The idea and structure of this FAQ is to introduce new terms along the way and build on them as to be not too confusing.

Please read and if there's any mistakes I'll change it. As I'm certain there are details that I missed or have gotten wrong.

Any additional questions that can be added to this are appreciated.

Also I hope the layout works as this is my first commit of this kind.

jcvernaleo commented:
Great, wanted to mention the filename stuff first but I'll give it a more detailed read.

jcvernaleo commented:
Also, your new page isn't linked to from anywhere.  It probably needs to go in the mkdocs.yml somewhere so it shows up in the site navigation.

jrick commented:
this new file needs a .md extension

jcvernaleo commented:
Yep, after the filename and link fixes @jrick and I can go over the content better.

animedow commented:
Ok, fixed the title and file extension.
I'll be honest and say I've got no clue how to add the file to the mkdocs.yml

jcvernaleo commented:
Cool, the name and the mkdocs stuff looks good.  I'll go through the text carefully tomorrow morning.

jcvernaleo commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+## **<i class="fa fa-book"></i> Links**
+
+[^1]: [Paymetheus Ticket Purchasing Guide] (/getting-started/user-guides/using-paymetheus.md#-purchase-tickets-tab)
```
You can't have a space between the anchor text and the link if you want it to render properly.  So no space between the ] and the (

jcvernaleo commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+#### **2. What are tickets?**
+
+A ticket is a token you buy to participate in the PoS system. You can buy tickets through the command line interface of dcrctl, or through a GUI like Paymetheus[^1] or Decredition. Whenever you buy a ticket you pay the current ticket price and a ticket fee using your DCR. Once your ticket is mined it will move from the current 'allmempool' to the 'immature' ticket pool. After 256 blocks (about 22 ...
```
It is spelled Decrediton.

jcvernaleo commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+The only cost for using this system are the fees you set. The txfee will be deducted for the transaction of your ticket across the network. The ticketfee will deducted for incorporation to the PoS pool and paid to the miners. If your ticket does not get mined the ticketfee will be returned to you along with the DCR paid for the ticket price.
+
+In case your ticket isn't selected to vote after 40960 blocks (about 4 months), the system will revoke your ticket and the DCR you paid for it will be transfered back into your wallet. You will not get the PoS reward. There is a low, about 2%, chance of this happening. The average time for a ticket to vote is 28 days.
```
Two rs in transferred

jcvernaleo commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+In case your ticket isn't selected to vote after 40960 blocks (about 4 months), the system will revoke your ticket and the DCR you paid for it will be transfered back into your wallet. You will not get the PoS reward. There is a low, about 2%, chance of this happening. The average time for a ticket to vote is 28 days.
+
+In case your ticket is selected to vote but your wallet cannot vote, eg. it isn't online or unlocked, the system marks the ticket as missed and the DCR you paid for it will be transferred back into your wallet after the system revokes your ticket. You will not get the PoS reward. To avoid this scenario the use of a stakepool is advised.
```
It is e.g.

jcvernaleo commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+By adding another parameter to the ticket that doesn't interfere with the approval of the mined blocks, the system can track the amount of tickets using that parameter over a number of blocks. You can set this parameter in your wallet at any time before the ticket votes. 
+
+For example, you could choose the colour of your ticket to be either red or blue, set that option, and the system will count for the next 1000 blocks how many red tickets and how many blue tickets there are. Maybe there are 3000 red tickets, 1500 blue tickets, and 500 that didn't select a colour.
```
I think we are standardized on US english spelling, so color.

jcvernaleo commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+Because the code for the hardfork is already in the then current version of Decred there is no need for the developers to interfere or the majority of PoW miners and PoS miners to upgrade after a decision has been reached. If the vote on a hardfork passes it will be implemented automatically after the lock in period.
+
+The hardfork vote can fail at multiple points. The vote might not start at all if the PoW miners or PoS miners do not upgrade. After that, the treshhold of 75% votes might not be reached.
```
threshold

gratefulcheddar commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+#### **1. What is Proof-of-Stake mining?**
+
+Besides Proof-of-Work (PoW) mining Decred has Proof-of-Stake (PoS) mining. In PoS mining the participants buy tickets using their Decred coins (DCR). These tickets go into the PoS pool and are randomly called upon to vote on a newly mined block. Every block 5 tickets are chosen from the pool and if at least 3 of those validate the mined block, the block is approved, added to the chain and both Po ...
```
I believe the tickets actually vote on whether the previous block was valid.

animedow replied:
I believe you are correct. They validate the last block so a new block can be mined/added to the chain.

animedow commented:
Updated with the minor changes jcvernaleo pointed out.
Changed the text about voting on/validation of the current block to say the previous block.

Are there any more content related comments?

jcvernaleo commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -104,16 +104,16 @@ For a hardfork vote to start two critical conditions have to be met:
 Because the code for the hardfork is already in the then current version of Decred there is no need for the developers to interfere or the majority of PoW miners and PoS miners to upgrade after a decision has been reached. If the vote on a hardfork passes it will be implemented automatically after the lock in period.
 
-The hardfork vote can fail at multiple points. The vote might not start at all if the PoW miners or PoS miners do not upgrade. After that, the treshhold of 75% votes might not be reached.
+The hardfork vote can fail at multiple points. The vote might not start at all if the PoW miners or PoS miners do not upgrade. After that, the treshold of 75% votes might not be reached.
```
Still not spelled correct.  threshold.

animedow replied:
I'm blind! Fixed.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+#### **1. What is Proof-of-Stake mining?**
+
+Besides Proof-of-Work (PoW) mining Decred has Proof-of-Stake (PoS) mining. In PoS mining the participants buy tickets using their Decred coins (DCR). These tickets go into the PoS pool and are randomly called upon to vote on the previous block. Every block 5 tickets are chosen from the pool and if at least 3 of those validate the previously mined block, the newly mined block is approved, added to ...
```
Need a comma right at the start:

> Besides Proof-of-Work (PoW),

jrick replied:
"PoS pool" is confusing wording and will be mistaken for stakepools.  The technical term is "ticket pool", although I admit there may be confusion with stakepools there as well.

jrick replied:
"and are randomly called upon"

This sounds like the ticket will be called upon eventually, while that's not the case.  There is a slim chance that the lottery won't choose the ticket at all, in which case it becomes expired and must be revoked.

I know you cover this later in detail but this intro needs to be a bit more accurate.

jrick replied:
> if at least 3 of those validate the previously mined block, the newly mined block is approved

This is inaccurate.  The voting is done on the previous block, and even with 3 no votes, the new block and non-stake transactions in it are still valid.

jrick replied:
Needs a comma:

> added to the chain, and both

animedow replied:
Right, I'll go over the text a bit more concerning grammar/punctuation.

On the points of the terms of the 'PoS mining' and 'PoS pool,' I used those because they're used in other sections of the docs too. I agree that they're confusing, especially the PoS pool one.

If we want to use different descriptions then we should also go over the other docs and change the terminology there.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,119 @@
+
+---
+
+#### **1. What is Proof-of-Stake mining?**
```
I probably wouldn't talk about "PoS mining" anywhere.  Mining refers to searching for a computationally hard answer through brute force, while our stake system doesn't have that.  Perhaps:

>####  1. How does Decred's Proof-of-Stake system work?

animedow commented:
Added jrick's changes to question 1.
Might need some more rewording/clarification.

Went over the other questions to clarify/reword the use of the term PoS pool and PoS miners.

Added that a platform for agenda setting is in development.

jcvernaleo commented:
@animedow was this waiting on changes from you or on further review?

fai04d commented:
I'd like to add my request for this to go through. I have some things to add to the doc when it's accepted.

animedow commented:
It's waiting on further review, that's why I've been letting it lay for the past couple of days hoping there would be some comments on anything beyond point 1.

But seeing as there haven't been any, I'm fine with going as is.

It's just that we might have to get the terminology for "PoS mining" or "PoS something" straight. But that can be done in edits too. I'll throw it in #documentation on slack too.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+Besides Proof-of-Work (PoW) mining, Decred has a Proof-of-Stake (PoS) system. In this PoS system the participants buy tickets using their Decred coins (DCR). These tickets go into the PoS system ticket pool and can be called upon to vote on the previous block, like a lottery. There is a very small chance your ticket will not be called upon to vote.
+
+Every block 5 tickets are chosen from the PoS ticket pool. If at least 3 of those validate the previously mined block, the newly mined block is added to the chain, and both PoW miners and PoS stakers are paid. If the block is not validated by the PoS stakers the PoW miners do not get paid, but the PoS stakers do. This is to incentivize PoW miners to mine according to the wishes of the PoS stakers ...
```
this still reads weirdly:

> If at least 3 of those validate the previously mined block, the newly mined block is added to the chain, 

While this is true that the block is added, the block is always added no matter which way the vote goes.  What the vote changes is whether the transactions in the previous block are logically removed from the chain (as opposed to "physical" removal where the block doesn't actually exist in the chain anywhere).

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+This PoS system has several advantages that solve problems PoW-only systems might encounter. For example, because the PoS stakers have to validate the blocks the PoW miners mine, the PoW miners cannot decide on their own to change the rules of the network (a 51% attack). Or, when the Decred chain hardforks the old, smaller, chain will die off quickly due to not being validated by the PoS componen ...
+
+The PoS system rewards participants for their staking. Every Decred block has a PoS reward component that will be distributed to every ticket chosen to vote in that block. The system is designed in such a way that the average time for the ticket to be selected is 28 days. If a ticket hasn't been selected after 40960 blocks (about 4 months) it will be revoked by the system. The DCR used to purchas ...
```
rewards are determined by whether the vote exists in the block, not when the ticket is chosen

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+The PoS system rewards participants for their staking. Every Decred block has a PoS reward component that will be distributed to every ticket chosen to vote in that block. The system is designed in such a way that the average time for the ticket to be selected is 28 days. If a ticket hasn't been selected after 40960 blocks (about 4 months) it will be revoked by the system. The DCR used to purchas ...
+
+Another interesting feature this PoS system allows for is the ability to vote on an agenda. Because the block validation can be considered a vote, by adding extra votebit components to the ticket the system can track and count votes over a series of blocks. These votes can be about anything.
```
i think this paragraph needs to be totally rewritten, it's just not clear enough as is.

For example:

> Besides voting on transactions in the previous block, the Decred PoS system can be used to vote on arbitrary decisions, including hard-forking consensus changes to the network.

This is a single sentence that covers essentially what this entire paragraph describes (without the technical details about bits that is not needed in this intro section) and also mentions how the system can be used for hardfork upgrades.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+A ticket is a token you buy to participate in the PoS system. You can buy tickets through the command line interface of dcrctl, or through a GUI like Paymetheus[^1] or Decrediton. Whenever you buy a ticket, you pay the current ticket price and a ticket fee using your DCR. Once your ticket is mined it will move from the current 'allmempool' to the 'immature' ticket pool. After 256 blocks (about 22 ...
+
+Remember, your wallet has to be online and unlocked for your ticket to cast a valid vote on a block when it is selected. If you don't want to leave your wallet unlocked and online, have a look at a PoS stake pool.
```
"Remember" can probably be striked from here, this is the first time in the document where we mention the wallet needing to be unlocked, so there's nothing to remember from before

jrick replied:
"have a look at" => "consider using"

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+#### **3. What is ticket price?**
+
+The price for tickets is determined by an algorithm. This algorithm aims to keep the ticket pool size, which is the total amount of tickets in the PoS system ticket pool, at around the target. The target is set to 40960 tickets. The ticket price will go up or down according to the demand for tickets.
```
I'd probably reword the first part to:

> The ticket price is determined by an algorithm that attempts to keep the number of live tickets close to the target value of 40960.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+The PoS system utilizes two types of fees, a txfee and a ticketfee.
+	
+The txfee is a fee you pay the network for handling the transaction to buy your ticket. This fee is at default set to a low amount (0.01 DCR/kB) and should not be changed.
```
"should not be" => "usually won't need to be"

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+The PoS system utilizes two types of fees, a txfee and a ticketfee.
+	
+The txfee is a fee you pay the network for handling the transaction to buy your ticket. This fee is at default set to a low amount (0.01 DCR/kB) and should not be changed.
+The ticketfee is the fee you pay if your ticket gets mined. This fee is an incentive for PoW miners to mine your ticket and add it to the ticket pool.
```
here i would mention that the "ticket fee" is also a transaction fee, but the "ticket fee" that you configure in the software only applies to ticket purchase transactions.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+#### **4. What are fees?**
+
+The PoS system utilizes two types of fees, a txfee and a ticketfee.
```
please refer to these as "split transaction fees" and "ticket purchase fee"

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+The txfee is a fee you pay the network for handling the transaction to buy your ticket. This fee is at default set to a low amount (0.01 DCR/kB) and should not be changed.
+The ticketfee is the fee you pay if your ticket gets mined. This fee is an incentive for PoW miners to mine your ticket and add it to the ticket pool.
+
+In case the buying window is full, meaning there are more tickets being bought than there are available to be mined during that window, the ticketfee will be used to determine which tickets get put into the ticket pool. The tickets with the highest ticketfee will be selected by the PoW miners.
```
This isn't guaranteed, it's just likely.  Instead, here i would say that increasing the ticket purchase fee gives incentive to PoW miners to include the ticket purchase in a block, and higher fees are more likely to be chosen when there is competition.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+In case the buying window is full, meaning there are more tickets being bought than there are available to be mined during that window, the ticketfee will be used to determine which tickets get put into the ticket pool. The tickets with the highest ticketfee will be selected by the PoW miners.
+
+Fees are calculated in DCR per kilobyte. Since a ticket is only about 300bytes in size, the actual fees you pay will be lower than what you set as the specific fee amount. Be aware, stake pool tickets are larger than solo-stake tickets.
```
300bytes => 300 bytes

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+Fees are calculated in DCR per kilobyte. Since a ticket is only about 300bytes in size, the actual fees you pay will be lower than what you set as the specific fee amount. Be aware, stake pool tickets are larger than solo-stake tickets.
+
+Both txfees and ticketfees will not be returned when a ticket votes, expires, or misses a vote.
```
need to mention that the original purchase price of the ticket will be returned,

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+Both txfees and ticketfees will not be returned when a ticket votes, expires, or misses a vote.
+
+Returning the funds back to your wallet after a ticket votes or is revoked does not generate a fee.
```
fees aren't generated, instead i would say "there is no fee associated with voting or revoking"

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+#### **5. What is a stake pool?**
+
+A stake pool is the equivalent of a mining pool, but for the PoS system. Through the options in your DCR wallet you can give your voting rights to a stake pool. If your ticket is selected to validate a block the stake pool will cast the vote for you and you are rewarded with the PoS reward. Unlike the mining pools, the PoS reward is not split amongst the users of the stake pool, but the full rewa ...
```
i don't think i would compare them to mining pools. in a mining pool, work and rewards are shared among all the participants. stakepools do not do either of these things. The only responsibility of the stakepool is to vote on your behalf, for a fee.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+A stake pool is the equivalent of a mining pool, but for the PoS system. Through the options in your DCR wallet you can give your voting rights to a stake pool. If your ticket is selected to validate a block the stake pool will cast the vote for you and you are rewarded with the PoS reward. Unlike the mining pools, the PoS reward is not split amongst the users of the stake pool, but the full rewa ...
+
+The use of a stake pool will allow you to buy tickets with your DCR but not have your own wallet unlocked and always online.
```
> Stake pools allow users to purchase tickets with voting rights delegated to a 3rd party to vote on their behalf.  This allows users who would otherwise be unable to run a 24/7 unlocked voting wallet to still participate in the PoS system.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+The use of a stake pool will allow you to buy tickets with your DCR but not have your own wallet unlocked and always online.
+
+Stake pools will usually implement multi-wallet redundancy by having many wallets physically distributed around the globe. This means there's less chance of a vote being missed because one wallet is down. It also reduces latency between the wallet and network which can reduce the chance of a vote being missed.
```
not usually, every official stakepool must prove redundancy

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+#### **6. What happens to my DCR?**
+
+So you successfully bought a ticket with your DCR. These coins will be locked into the PoS ticket pool	until the ticket is selected to validate a block or it expires. When your coins are in the PoS ticket pool they are locked and safe, nothing can happen to them. If your ticket is selected to validate a block you will be refunded the total amount of your DCR you paid for the ticket and, as a rewa ...
```
> So you successfully bought a ticket with your DCR.

I did?

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+So you successfully bought a ticket with your DCR. These coins will be locked into the PoS ticket pool	until the ticket is selected to validate a block or it expires. When your coins are in the PoS ticket pool they are locked and safe, nothing can happen to them. If your ticket is selected to validate a block you will be refunded the total amount of your DCR you paid for the ticket and, as a rewa ...
+
+The only cost for using this system are the fees you set. The txfee will be charged for the transaction of your ticket across the network. The ticketfee will be charged for incorporation into the PoS ticket pool, and is paid to the miners. If your ticket does not get mined, in case of a full buying window, the ticketfee will be returned to you along with the DCR paid for the ticket price.
```
> The txfee will be charged for the transaction of your ticket across the network. 

what?

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+Because the block validation performed by the PoS system behaves like a voting system, it can be used to vote on other issues too.
+
+When selected to validate a block, the tickets vote on whether or not to approve the previously mined block. This requires 3 out of 5 selected tickets to vote yes.
```
tickets don't vote, votes vote.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+When selected to validate a block, the tickets vote on whether or not to approve the previously mined block. This requires 3 out of 5 selected tickets to vote yes.
+
+By adding another parameter to the ticket that doesn't interfere with the approval of the mined blocks, the system can track the amount of tickets using that parameter over a number of blocks. You can set this parameter in your wallet at any time before the ticket votes. 
```
again, not tickets, votes.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+By adding another parameter to the ticket that doesn't interfere with the approval of the mined blocks, the system can track the amount of tickets using that parameter over a number of blocks. You can set this parameter in your wallet at any time before the ticket votes. 
+
+For example, you could choose the color of your ticket to be either red or blue, set that option, and the system will count for the next 1000 blocks how many red tickets and how many blue tickets there are. Maybe there are 3000 red tickets, 1500 blue tickets, and 500 that didn't select a colour.
```
besides the mixing of the spellings color and colour here, i don't even see the point of this example.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+Like any other cryptocurrency Decred might need to hardfork at some point.
+
+One of the agenda issues set for the PoS voting system could be a hardfork. If such an issue is set, the new version of the decred files will include the hardfork but it will not activate until the PoS system has voted on it.
```
files => software

jrick replied:
and a vote must succeed with a majority for the change to be locked in and then activated.

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+- First, 75% of the PoW miners have to upgrade to the latest version of the current network block version. This check runs over the past 100 blocks.
+- Second, 75% of the PoS miners have to upgrade to the latest version. This check runs over the past 2016 blocks.
+
+If these checks are met the voting process begins. Tickets can be set with an additional parameter of yes/no/abstain. You do this in your wallet before the ticket votes. Tickets marked as abstain will not be counted in the total amount of votes.
```
votes not tickets

jrick commented on docs/faq/proof-of-stake/pos-general-faq.md:
```diff
@@ -0,0 +1,123 @@
+
+If these checks are met the voting process begins. Tickets can be set with an additional parameter of yes/no/abstain. You do this in your wallet before the ticket votes. Tickets marked as abstain will not be counted in the total amount of votes.
+
+The PoS system then starts counting tickets with these parameters set over a predetermined amount of blocks. If over this interval the amount of votes is 75%, or more, 'yes' the vote has passed. A lock in period of a set number of blocks will start before the hardfork goes live, so everyone has a chance to upgrade and not be hardforked off the network.
```
counting tickets => counting votes

jrick replied:
> If 75% or more of the votes vote for the consensus change, the vote passes.

animedow commented:
Thank you for your many comments jrick. I was hoping someone would comment on it to point out the factual errors so I can change them.

As for the formatting/wording of the guide. The idea was that it would be a very low level FAQ document answering the kind of questions we get in the slack. In my opinion there's two ways we can go about this: 1. Keep the answers low brow and explain in wordy terms how things work. 2. Be highly efficient and technical.

Obviously, I chose the first option, as most people who have been asking these kind of questions have shown no affinity for any kind of these systems. The second option just confuses these people, have them answer a generic "hmm, yes", forget about it and either misinform others or walk away. If people have more technical questions or want to learn how things really work, there should be a technical faq or document that explains these things. This, however, is no excuse for there to be factual errors in the general faq.

Also, I'm reverting the changes to the terms regarding the PoS system, to have it read 'mining' again. As everything else on both the website, docs, and even jy-p's recent coinbase talk, talk about PoS mining and PoS miners. (due to mining being defined as validating blocks and transactions.) If there is consensus on it being changed globally this document can be changed in that sweep, too.

jcvernaleo commented:
@animedow and @jrick where are we on this on?

jcvernaleo commented:
This PR is now significantly behind master.  What is the status of it?  Is it even still relevant (since docs have been moving pretty quickly)?
@animedow @jrick ?
If it is still possible to fix I'd like it rebased and then we can see what needs to be done.  If it is dead then I'd like to close it.

fai04d commented:
If @animedow isn't working on the docs anymore I'll grab the file and keep going (I guess it'll need to be a new PR?). I have a few things to add to it.

jcvernaleo commented:
I'll ping @animedow on slack.  If I don't hear back in a day or two then you can take over this PR @fai04d.  Best way would be for you to grab the file (or the commit even), I close this one, and you open a new one.  And of course we credit @animedow's contribution in the commit message for the new one.  But let me check slack first.

jcvernaleo commented:
@fai04d I think we've gone long enough so feel free to take this one over.  Either grab the file or commit from this and then I'll close this one.  Just make sure to credit @animedow in your commit for the work that was already done.
